src/BusHelper.js

```diff
@@ -97,6 +97,13 @@ class BusHelper extends EventEmitter {
     return this._ifaceProxy[methodName](...args)
   }
 
+  removeListeners () {
+    this.removeAllListeners('PropertiesChanged')
+    if (this._propsProxy !== null) {
+      this._propsProxy.removeAllListeners('PropertiesChanged')
```

Comment:
@tuxedoxt  I understand that you want to delete this listener because it generate a memory leak, but I think that if you do this in this way, you creates a different bug. 
What if the user connects again this device? No code line will subscribe again on 'PropertiesChanged' events, so the library will not work correctly.

I think that this bug requires a different fix.

What do you think?

Reply:
Thanks for the feedback. I see what you mean.

As I see it, additionally setting

```
this._ready = false
```

should make `_prepare` reattach listener. However also does other things, like assigns completely new `_propsProxy` and `_ifaceProxy` objects. Would more clean-up be needed there or is this feasible?